backend/python/app/graphql/mutations/auth_mutation.py

```diff
@@ -4,4 +4,41 @@
     refresh: String!
     logout(userId: ID!): ID
     resetPassword(email: String!): Boolean!
- """
+"""
+
+import os
+
+import graphene
+from flask import Blueprint, current_app, jsonify, request
+
+from ...services.implementations.auth_service import AuthService
+from ...services.implementations.email_service import EmailService
+from ..service import services
+
+email_service = EmailService(
+    current_app.logger,
+    {
+        "refresh_token": os.getenv("EMAIL_REFRESH_TOKEN"),
+        "token_uri": "https://oauth2.googleapis.com/token",
+        "client_id": os.getenv("EMAIL_CLIENT_ID"),
+        "client_secret": os.getenv("EMAIL_CLIENT_SECRET"),
+    },
+    "planetread@uwblueprint.org",  # must replace
+    "Planet Read",  # must replace)
+)
+auth_service = AuthService(current_app.logger, services["user"], email_service)
+
+
+class ResetPassword(graphene.Mutation):
+    class Arguments:
+        email = graphene.String(required=True)
+
+    ok = graphene.Boolean()
+
+    def mutate(root, info, email):
+        try:
```

Comment:
I think this try statement is unnecessary. An error raised in `auth_service.reset_password(email) ` will continue to be raised down the call stack, so I don't think you need to re-raise this `except`

Reply:
Hmm this is what the REST endpoint looks like, what should this be? Only keeping lines 40-41 without a `try` wrapping?

Reply:
huh. You're right, that is how the REST endpoint does it. Feel free to keep the same code, I'm going to ask internal tools about this choice.